client/src/components/GroupCard/GroupCard.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import "./GroupCard.css";
+import { Link } from "react-router-dom";
+
+const GroupCard = (props) => {
+
+  let background;
+
+  if (props.url) {
+    background = <img className="GroupCard__background-image" src={props.url} alt="group background" />;
+  } else {
+    background = "";
+  }
```

Comment:
You could try using the `background-image` CSS property here (set it with the `style` attribute). That may clean this up a lot

Reply:
@shelbyvjacobs Did you forget this one?